executor/builder.go

```
@@ -901,3 +903,56 @@ func (b *executorBuilder) buildIndexReader(v *plan.PhysicalIndexReader) Executor
 	b.err = e.doRequest()
 	return e
 }
+
+func (b *executorBuilder) buildIndexLookUpReader(v *plan.PhysicalIndexLookUpReader) Executor {
+	dagReq := &tipb.DAGRequest{}
+	dagReq.StartTs = b.getStartTS()
+	dagReq.TimeZoneOffset = timeZoneOffset()
+	sc := b.ctx.GetSessionVars().StmtCtx
+	dagReq.Flags = statementContextToFlags(sc)
+	for _, p := range v.IndexPlans {
+		execPB, err := p.ToPB(b.ctx)
+		if err != nil {
+			b.err = errors.Trace(err)
+			return nil
+		}
+		dagReq.Executors = append(dagReq.Executors, execPB)
+	}
+
+	tableReq := &tipb.DAGRequest{
+		StartTs:        dagReq.StartTs,
+		TimeZoneOffset: dagReq.TimeZoneOffset,
+		Flags:          dagReq.Flags,
+	}
+	for _, p := range v.TablePlans {
+		execPB, err := p.ToPB(b.ctx)
+		if err != nil {
+			b.err = errors.Trace(err)
+			return nil
+		}
+		tableReq.Executors = append(tableReq.Executors, execPB)
+	}
+
+	is := v.IndexPlans[0].(*plan.PhysicalIndexScan)
+	table, _ := b.is.TableByID(is.Table.ID)
```

We must find it. It's ported from old old. This error can be ignored.